Tabloid/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using Tabloid.Models;
+using Tabloid.Repositories;
+
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
Should this controller be auth protected? should there be auth checks in any of the methods?

Reply:
auth protected controller, will add auth checks to methods to verify user type permissions when that feature is added